web-wallet/CHANGELOG.md

```diff
@@ -142,9 +144,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [#1469]: https://github.com/dusk-network/rusk/issues/1469
 [#1470]: https://github.com/dusk-network/rusk/issues/1470
 [#1471]: https://github.com/dusk-network/rusk/issues/1471
-[#1362]: https://github.com/dusk-network/rusk/issues/1362
```

Comment:
Problem with the conflict resolution – this shouldn't have been removed.

@kieranhall